packages/mds-api-authorizer/index.ts

```diff
@@ -10,17 +10,17 @@ export interface AuthorizerClaims {
   jurisdictions: string | null
 }
 
-const {
-  TOKEN_PROVIDER_ID_CLAIM = 'https://ladot.io/provider_id',
-  TOKEN_USER_EMAIL_CLAIM = 'https://ladot.io/user_email',
-  TOKEN_JURISDICTIONS_CLAIM = 'https://ladot.io/jurisdictions'
-} = process.env
-
 export type Authorizer = (authorization: string) => AuthorizerClaims | null
 export type ApiAuthorizer = (req: express.Request) => AuthorizerClaims | null
 
 const decoders: { [scheme: string]: (token: string) => AuthorizerClaims } = {
   bearer: (token: string) => {
+    const {
+      TOKEN_PROVIDER_ID_CLAIM = 'https://ladot.io/provider_id',
```

Comment:
FWIW we ought to not have these defaults for the contribution

Reply:
I agree. I think they should be `https://openmobilityfoundation.org/provider_id`, etc, but we would have to make sure the environment variables are set in environments that are using the present defaults before the changes were deployed. We should actually be setting them explicitly for _all_ environments anyway and not relying on the default values at all.